raw_package/static/css/esphome-2.css

```diff
@@ -15,11 +14,50 @@
   --alert-warning-color-bg: #fff5e6;
   --alert-error-color: #d93025;
   --alert-error-color-bg: #faefeb;
-
-  --mdc-theme-primary: #03a9f4;
+  --mdc-theme-primary: #212121;
```

Comment:
I think it's a mistake. Instead, we should just override the button to have the theme set to the color that we want (`--primary-text-color`). If we ever want to use the theme primary on the material components, we wouldn't want to be stuck with black and it being a huge breaking change.

Reply:
Have reverted the root `--mdc-theme-primary`